src/mgr/MgrCommands.h

```diff
@@ -5,6 +5,8 @@
 
 // see MonCommands.h
 
+COMMAND("get_command_descriptions", "list commands descriptions", "mgr", "r", "cli,rest")
```

Comment:
emm...keep it simple since we will have `tell module help`. we could add it if user need it.